cumulusci/tasks/metadata_etl/help_text.py

```diff
@@ -37,7 +50,7 @@ def _init_options(self, kwargs):
             )
         if not all(["api_name" in entry for entry in self.options["fields"]]):
             raise TaskOptionsError(
-                "The 'api_name' key is required on all entry values."
+                "This option is auto-populated based on the 'fields' option. It represents a list of API names of entities to affect. This option is not intended to be directly provided by the user."
```

Comment:
@Julian88Tex I think this change is misplaced. I see what you're aiming to do but throwing an error if `api_names` is populated would need to be before line 37. This error gets thrown if the dicts are not structured correctly.

(And we should really revise how this whole thing works, but that's a topic for another day).

Reply:
@davidmreed oops yeah no idea what I was thinking there. Looks like the description is actually in the base.py file: https://github.com/SFDO-Tooling/CumulusCI/blob/e5e9d8b2341b644d0b1e1284513b613c8445d3ab/cumulusci/tasks/metadata_etl/base.py#L250

So yeah doesn't make sense to change it there but probably remove showing this option altogether, but yeah probably ab bigger change and out-of-scope for the intention of this change.

Removed it and updated :)